src/store/entities.js

```diff
@@ -52,7 +52,7 @@ export const defaultColumns = (groupsEnabled = false) => ([
         title: 'Group',
         props: { width: 10, isStatic: true },
         // eslint-disable-next-line camelcase
-        renderFunc: (groups) => isEmpty(groups) ? 'N/A' : groups[0].name, // currently, one group at maximum is supported
+        renderFunc: (groups) => isEmpty(groups) ? 'N/A' : groups[0]?.name, // currently, one group at maximum is supported
```

Comment:
Is this change really needed? If the groups are empty, then 'N/A' will be rendered. Otherwise, we expect that groups contain the data with valid format (here means, containing `name`).

Reply:
Agree, removed the conditional chaining